### PR TITLE
Update /partners/desktop

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Updated [/partners/desktop](https://canonical-com-889.demos.haus/partners/desktop) according to [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9447-23217&t=iTAW7GdzlqIpWZ1g-0)

## QA

- Compare [/partners/desktop](https://canonical-com-889.demos.haus/partners/desktop) against [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9447-23217&t=iTAW7GdzlqIpWZ1g-0)

## Issue / Card

Fixes WD-3739

## My two cents
This doesn't really feel like an upgrade. None of the aligments quite work, there are too many horizontal lines it gets distracting.
All in all, it feels a bit dull and wonky in my opinion (who no one asked for)

### CODE REVIEW

File: templates/partners/desktop.html [hunk 1]
```diff
@@ -6,144 +6,243 @@
...
+    <div class="col">
+      <p>Security, compliance and support to improve efficiency while reducing complexity and cost.</p>
+      <hr>
+      <p><a href="https://buy.ubuntu.com/">Learn about our support options</a></p>
+    </div>
+  </div>
+</section>
+
+<style>
+  .long-white-rectangle{
```
Comment (lyubomir-popov) [inline templates/partners/desktop.html:237]: Remember the value of 1rem changes at a breakpoint (1681px IIRC), so anything other than 1px or 3px should not be expressed in px, but rems. Separately, The idea is to have 2x3 aspect reatio, I would base it arouind that - and ensure there's padding around the logos. The screenshot below lacks some padding around the lenovo logo: 
image
)

File: templates/partners/desktop.html [hunk 2]
```diff
@@ -6,144 +6,243 @@
...
+    height: 420px;
```
Comment (lyubomir-popov) [inline templates/partners/desktop.html:238]:
```suggestion
    aspect-ratio: .67
```
should look like so:
image

File: templates/partners/desktop.html [hunk 3]
```diff
@@ -6,144 +6,243 @@
...
-      </div>
+
+<div class="row">
+  <div class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
+    <ol class="p-breadcrumbs__items p-text--small-caps">
+      <li class="p-breadcrumbs__item "><a href="/partners">Partners</a></li>
+      <li class="p-breadcrumbs__item ">Desktop</li>
+    </ol>
+  </div>
+  <hr>
```
Comment (lyubomir-popov) [inline templates/partners/desktop.html:20]: all hrs in the new designs should be p-rules, unless there's a button immediately above them
```suggestion
  <hr class="p-rule">
```

**Review by lyubomir-popov — CHANGES_REQUESTED** [review]
Quite close, a few small comments  in figma here. I think we have a bug with the logo-section within these new layouts. As an overall comment, I would avoid using the new layouts, in my experience they cause more trouble than  they help.

**Review by lyubomir-popov — CHANGES_REQUESTED** [review]
A few things need fixing here, but overall looks really good, thanks @ClementChaumel https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9965%3A24531&t=iDY29WLvf5WCb1hc-1

File: templates/partners/desktop.html [hunk 4]
```diff
@@ -6,144 +6,243 @@
...
+  </div>
+  <div class="row--50-50">
+    <div class="col">
+      <h2>A selection of our Desktop partners</h2>
+    </div>
+    <div class="col">
+      <ul class="p-logo-section__items" style="justify-content: space-around;">
+        {% for partner in partners %}
+        <li class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
```
Comment (mtruj013) [inline templates/partners/desktop.html:47]: Not in scope for this pr I think, but it'd be great if we could update these. The Dell and Lenovo logos look a bit too small

File: templates/partners/desktop.html [hunk 5]
```diff
@@ -6,144 +6,243 @@
...
+        </li>
+        {% endfor %}
+      </ul>
+      <hr>
+      <p class="u-align--right"><a href="/partners/find-a-partner?filters=desktop">See all desktop partners</a></p>
```
Comment (mtruj013) [inline templates/partners/desktop.html:52]: Are links supposed to be followed by a chevron @lyubomir-popov? I've seen some with and some without on a few mockups, so I'm not sure

File: templates/partners/desktop.html [hunk 6]
```diff
@@ -6,144 +6,243 @@
...
+    <div class="col">
       <h2>Buy a certified Ubuntu PC</h2>
-      <p>We've been working with partners like Dell, HP and Lenovo for years to ensure that Ubuntu devices reach market as effectively as possible. With the help of these partners, Ubuntu machines are distributed around the world and can be found in both their exclusive and partner stores.</p>
     </div>
-    <div class="row">
-      <div class="col-4 p-card--highlighted has-cta">
-        <div class="p-card__content">
-          <img src="https://assets.ubuntu.com/v1/d258d23b-2018-logo-dell.svg" alt="Dell" width="80" height="80">
+    <div class="col">
+      <p>We've been working with partners like Dell, HP and Lenovo for years to ensure that Ubuntu devices reach market as effectively as possible. With the help of these partners, Ubuntu machines are distributed around the world and can be found in both their exclusive and partner stores.</p>
```
Comment (mtruj013) [inline templates/partners/desktop.html:178]: Looks like these sentences are each supposed to be a separate p

**Review by mtruj013 — APPROVED** [review]
Thanks @ClementChaumel ! Just a few minor suggestions. The footer doesn't match the design but I understand we're waiting for content to be finalized before we make copy changes

**Comment by ClementChaumel** [comment]
> Thanks @ClementChaumel ! Just a few minor suggestions. The footer doesn't match the design but I understand we're waiting for content to be finalized before we make copy changes

The footer is being updated part of this one: https://github.com/canonical/canonical.com/pull/886 since it's a partial I thought doing it in one place should be enough!

**Comment by ClementChaumel** [comment]
@lyubomir-popov I'd like to avoid using utility classes as much as possible because I think they convey that components are not fit for real-world use and if we start using the brochure layout this way it'll never get used right.

I think it'd be better to update the components/patterns than fix them here with utility classes. Or at the very least update the doc to mention that usage of those utilities is expected and best practice.

Here's what you're proposing:
image

```html
          <hr class="rule">
          <p class="u-no-margin--bottom">
            <a href="https://www.delltechnologies.com/en-gb/ai-technologies/index.htm">See the Ubuntu certified AI-ready workstations</a>
          </p>
          <hr class="rule">
          <p class="u-no-margin--bottom">
            <a href="http://www.dell.com/linux">Shop for Dell with Ubuntu</a>
          </p>
```

Would you be ok with this alternative?

image


```html
          <hr>
          <a href="https://www.delltechnologies.com/en-gb/ai-technologies/index.htm">See the Ubuntu certified AI-ready workstations</a>
          <hr>
          <a href="http://www.dell.com/linux">Shop for Dell with Ubuntu</a>
```

keeping the markup leaner and using the default `<hr>` for spacing instead of adding a `<p>` just for its top spacing.

File: templates/partners/desktop.html (hunk 4 above)
Comment (ClementChaumel) [reply templates/partners/desktop.html:48]: Agreed, do you know who is in charge of this?

**Comment by ClementChaumel** [comment]
@lyubomir-popov I don't think that if the goal is to add spacing between the line and the link this:
image
is the best way. Especially when we can achieve something almost identical with less doing and undoing (Occam's razor, KISS, etc.)

This is the difference
spacing2

**Comment by lyubomir-popov** [comment]
@ClementChaumel I'll finish that one.

File: templates/partners/desktop.html [hunk 7]
```diff
@@ -6,144 +6,245 @@
...
+  </div>
+  <div class="row--50-50">
+    <div class="col">
+      <h2>A selection<br class=u-hide--small> of our Desktop partners</h2>
+    </div>
+    <div class="col">
+      <ul class="p-logo-section__items" style="justify-content: space-around;">
+        {% for partner in partners %}
+        <li class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
```
Comment (lyubomir-popov) [inline templates/partners/desktop.html]: @ClementChaumel what is this for? Is it so cdns don't mangle the images? `style="min-width: 5rem;" `

**Comment by lyubomir-popov** [comment]
@ClementChaumel there are some visual changes that Marcus suggested (laptops instead of logos), just a heads up.

**Review by lyubomir-popov — APPROVED** [review]

**Comment by juanruitina** [comment]
Only one comment: on small viewports, I would feel inclined to have the items under "Why buy a certified Ubuntu desktop?" take the full width of the page, instead of having two columns (one for title, second for description). @lyubomir-popov would you agree with this?

**Comment by ClementChaumel** [comment]
> Only one comment: on small viewports, I would feel inclined to have the items under "Why buy a certified Ubuntu desktop?" take the full width of the page (see screenshot, first two items), instead of having two columns (one for title, second for description, last two items in the screenshot). @lyubomir-popov would you agree with this?
> 
> canonical-com-889 demos haus_partners_desktop(Figma mobile)

That's the way it was before, I'll revert it!

**Comment by ClementChaumel** [comment]
Done, @juanruitina please switch the label form UX needed to UX+1 if you're happy with the changes!

**Comment by juanruitina** [comment]
LGTM! (We will harmonise the links at the bottom when we do the general QA)

**Pull request merged**